firmware/rhizome.ino

```diff
@@ -7,6 +7,7 @@
 #include "Ohmbrewer_Publisher.h"
 #include "Ohmbrewer_Thermostat.h"
 #include "Ohmbrewer_RIMS.h"
+#include "Ohmbrewer_Onewire.h"
 #include "onewire.h"
```

Comment:
Until such time that we don't use an std::list in rhizome.ino, I'd put it in there. It doesn't hurt, at the very least.